CF01/email-validations.js

```diff
@@ -0,0 +1,31 @@
+const { expect } = require("chai");
+const { beforeEach } = require("mocha");
+
+describe ('Email Errors', function(){
+    beforeEach(function() {
+        cy.visit('https://qa-angular-conduit.vercel.app/register'); 
```

Comment:
Add base url in cypress.json file.
https://docs.cypress.io/guides/references/configuration.html#Options
And in this line, only use `cy.visit('register');`